indico/modules/events/contributions/templates/management/contributions.html

```diff
@@ -10,16 +10,16 @@
     {%- if event.type == 'conference' -%}
         <a id="contributions-settings-dropdown"
            class="i-button borderless icon-settings arrow js-dropdown"
-           data-toggle="dropdown"></a>
+           data-toggle="dropdown">Settings</a>
```

Comment:
Looks a bit weird to me (also, missing i18n)...

![](https://cloud.githubusercontent.com/assets/179599/20703331/9a4888a8-b61b-11e6-9faf-a679541eab80.png)